tests/steps/test_cases_functional.py

```diff
@@ -0,0 +1,195 @@
+# Author: Laurentiu Mandru
+# Email: mclaurentiu79@gmail.com
+
+# coding=utf-8
+"""Functional test cases for LAM - steps implementation"""
+import logging
+from time import sleep
+
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+    parsers
+)
+
+logger = logging.getLogger(__name__)
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.01 – Access LAM online tool')
+def test_tc01__access_lam_online_tool():
+    """TC.01 – Access LAM online tool."""
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.03 – TOC for Properties - Expands and collapses')
+def test_tc03_toc_for_properties_expands_and_collapses():
+    """TC.03 – TOC for Properties - Expands and collapses"""
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.03 – TOC for Properties - Selected entry is highlighted')
+def test_tc03_toc_for_properties_selected_entry_is_highlighted():
+    """TC.03 – TOC for Properties - Selected entry is highlighted"""
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.04 – TOC for Templates - Expands and collapses')
+def test_tc04_toc_for_templates_expands_and_collapses():
+    """TC.04 – TOC for Templates - Expands and collapses"""
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.05 – TOC for Celex numbers - Expands and collapses')
+def test_tc05_toc_for_celex_numbers_expands_and_collapses():
+    """TC.05 – TOC for Celex numbers - Expands and collapses"""
+
+
+@scenario('../features/functional_test_cases.feature', 'TC.05 – TOC for Celex numbers - Selected entry is highlighted')
+def test_tc05_toc_for_celex_numbers_selected_entry_is_highlighted():
+    """TC.05 – TOC for Celex numbers - Selected entry is highlighted"""
+
+
+@scenario('../features/functional_test_cases.feature', 'Verify that specific properties exist')
+def test_verify_that_specific_properties_exist():
+    """Verify that specific properties exist"""
+
+
+@scenario('../features/functional_test_cases.feature', 'Verify that specific classes exist')
+def test_verify_that_specific_classes_exist():
+    """Verify that specific classes exist"""
+
+
+@scenario('../features/functional_test_cases.feature', 'Verify that specific CELEXs exist')
+def test_verify_that_specific_celexs_exist():
+    """Verify that specific CELEXs exist"""
+
+
+@given(parsers.cfparse('the baseURI {base_uri:String}', extra_types=dict(String=str)))
+def the_baseuri_lam_uri(browser, scenario_context, base_uri):
+    """the baseURI LAM_URI."""
+    if base_uri == "LAM_URL":
+        from tests.config import LAM_URL
+        scenario_context["base_uri"] = LAM_URL
+
+
+@when(parsers.cfparse('I navigate to the location {pageLocation:String}', extra_types=dict(String=str)))
+def i_navigate_to_the_location_page(scenario_context, browser, pageLocation):
+    browser.get(scenario_context["base_uri"] + pageLocation)
+
+
+@then(parsers.cfparse('the resulting page contains {content:String} in the element with id {field_id:String}',
```

Comment:
it doesn't seem that there are `feature` steps with this text, consider removing them?